compiler/shared/test/diff-ir/IR.mls

```diff
@@ -14,28 +15,9 @@ foo()
 //│ 
 //│ 
 //│ IR:
-//│ Program({ClassInfo(0, Pair, [x,y])}, {
-//│ Def(0, mktup2, [x$0,y$0],
-//│ 1, 
-//│ let* (x$1) = mktup(x$0,y$0) in -- #7
-//│ x$1 -- #6
-//│ )
-//│ Def(1, mktup, [x$2,y$1],
-//│ 1, 
-//│ let x$3 = Pair(x$2,y$1) in -- #14
-//│ x$3 -- #13
-//│ )
-//│ Def(2, foo, [],
-//│ 1, 
-//│ let* (x$4) = mktup2(1,2) in -- #22
-//│ x$4 -- #21
-//│ )
-//│ },
-//│ let* (x$5) = foo() in -- #26
-//│ x$5 -- #25)
```

Comment:
Why is this no longer printed? And if that's intended, shouldn't the "IR:" part also be removed?

Reply:
It is intended (see https://github.com/hkust-taco/mlscript/pull/218#discussion_r1598011027); I forgot to remove the `IR:` printout.